### PR TITLE
Bach docs: add new Series

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds docs for the new Bach Series types, as well as redirects from old URLs.

Redirects:

- /docs/modeling/Series/bach.SeriesAbstractDateTime.dt          -> /docs/modeling/bach/api-reference/Series/
- /docs/modeling/Series/bach.SeriesAbstractDateTime             -> /docs/modeling/bach/api-reference/Series/
- /docs/modeling/bach/api-reference/Series/AbstractDateTime     -> /docs/modeling/bach/api-reference/Series/
- /docs/modeling/bach/api-reference/Series/AbstractDateTime/dt  -> /docs/modeling/bach/api-reference/Series/
- /docs/modeling/bach/api-reference/Series/Jsonb      -> /docs/modeling/bach/api-reference/Series/Json
- /docs/modeling/bach/api-reference/Series/Jsonb/json -> /docs/modeling/bach/api-reference/Series/Json

Changes from this PR: https://github.com/objectiv/objectiv-analytics/pull/742

### CODE REVIEW

File: .htaccess-production [hunk 1]
```diff
@@ -105,8 +105,21 @@ RewriteRule ^.*?$ /docs/modeling/bach/api-reference/DataFrame/ [L,R=301]
...
+RewriteCond %{REQUEST_URI} ^/(docs/modeling/Series/bach.SeriesAbstractDateTime)(\/?)$
+RewriteRule ^.*?$ /docs/modeling/bach/api-reference/Series/Date/ [L,R=301]
+# /docs/modeling/bach/api-reference/Series/AbstractDateTime     -> /docs/modeling/bach/api-reference/Series/Date
+# /docs/modeling/bach/api-reference/Series/AbstractDateTime/dt  -> /docs/modeling/bach/api-reference/Series/Date
+RewriteCond %{REQUEST_URI} ^/(docs/modeling/bach/api-reference/Series/AbstractDateTime)
+RewriteRule ^.*?$ /docs/modeling/bach/api-reference/Series/Date/ [L,R=301]
+# /docs/modeling/bach/api-reference/Series/Jsonb      -> /docs/modeling/bach/api-reference/Series/Json
+# /docs/modeling/bach/api-reference/Series/Jsonb/json -> /docs/modeling/bach/api-reference/Series/Json
+RewriteCond %{REQUEST_URI} ^/(docs/modeling/bach/api-reference/Series/Jsonb)
+RewriteRule ^.*?$ /docs/modeling/bach/api-reference/Series/Json/ [L,R=301]
```
Comment (ivarpruijn) [inline .htaccess-production:121]: @thijs-obj Could you carefully check if these redirects make sense please? Thanks!

File: blog/2022-03-17 Release v0.0.14 Extended Bach date-time operations.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ wrangling/exploring time-related data.*
 Like any Bach operation, they work on the full SQL dataset.
 
 A few highlights of the features that we’ve added:
-- [`TimeDelta`](https://objectiv.io/docs/modeling/bach/api-reference/Series/AbstractDateTime/dt/) numeric: 
+- [`TimeDelta`](https://objectiv.io/docs/modeling/bach/api-reference/Series/Timedelta/): 
 Get days/seconds/microseconds/components between two time series.
```
Comment (ivarpruijn) [inline blog/2022-03-17 Release v0.0.14 Extended Bach date-time operations.md:26]: @thijs-obj This change also makes sense, right?

File: docs/docs/modeling/example-notebooks/open-taxonomy.mdx [hunk 3]
```diff
@@ -70,7 +70,7 @@ The location stack and global contexts are stored as json type data. Within the
 **Slicing the json data**
 With the `.json[]` syntax you can slice the array using integers. Instead of integers, dictionaries can also be passed to ‘query’ the json array. If the passed dictionary matches a context object in the stack, all objects of the stack starting (or ending, depending on the slice) at that object will be returned.
 
-In case a json array does not contain the object, `None` is returned. More info at the [API reference](/modeling/bach/api-reference/Series/Jsonb/index.mdx).
+In case a json array does not contain the object, `None` is returned. More info at the API reference.
```
Comment (ivarpruijn) [inline docs/docs/modeling/example-notebooks/open-taxonomy.mdx:73]: Currently broken, working on a fix

File: docs/docs/modeling/example-notebooks/open-taxonomy.mdx [hunk 4]
```diff
@@ -90,7 +90,7 @@ the column. The methods are:
       * [.ls.nice_name](/modeling/open-model-hub/api-reference/SeriesLocationStack/modelhub.SeriesLocationStack.ls.mdx#ls-nice-name)
 
 
-* all [methods](/modeling/bach/api-reference/Series/Jsonb/index.mdx#json-accessor) for the json(b) type can also be accessed using `.ls`
+* all methods for the json(b) type can also be accessed using `.ls`
```
Comment (ivarpruijn) [inline docs/docs/modeling/example-notebooks/open-taxonomy.mdx:93]: Currently broken, working on a fix

File: docs/docs/modeling/example-notebooks/open-taxonomy.mdx [hunk 5]
```diff
@@ -120,7 +120,7 @@ The `global_contexts` column in the DataFrame contain all information that is re
       * [.gc.application](/modeling/open-model-hub/api-reference/SeriesGlobalContexts/modelhub.SeriesGlobalContexts.global_contexts.mdx#gc-application)
 
 
-* all [methods](/modeling/bach/api-reference/Series/Jsonb/index.mdx#json-accessor) for the json(b) type can also be accessed using `.gc`
+* all methods for the json(b) type can also be accessed using `.gc`
```
Comment (ivarpruijn) [inline docs/docs/modeling/example-notebooks/open-taxonomy.mdx:123]: Currently broken, working on a fix

File: .htaccess-production (hunk 1 above)
Comment (thijs-obj) [reply .htaccess-production:121]: I think we could skip these redirects:

```
/docs/modeling/Series/bach.SeriesAbstractDateTime.dt -> /docs/modeling/bach/api-reference/Series/Timedelta
/docs/modeling/Series/bach.SeriesAbstractDateTime -> /docs/modeling/bach/api-reference/Series/Date
/docs/modeling/bach/api-reference/Series/AbstractDateTime -> /docs/modeling/bach/api-reference/Series/Date
/docs/modeling/bach/api-reference/Series/AbstractDateTime/dt -> /docs/modeling/bach/api-reference/Series/Date
```

They are a bit arbitrary. We could redirect `SeriesAbstractDateTime` to `SeriesTime`, to `SeriesDate`, to `SeriesTimeStamp`, or to `SeriesTimedelta`. All four options make some sense. Now in the above we do two of those four arbitrarily.

I think it makes more sense to either 1) just not redirect and leave the links broken. OR 2) redirect to `docs/modeling/bach/api-reference/Series/`

File: blog/2022-03-17 Release v0.0.14 Extended Bach date-time operations.md (hunk 2 above)
Comment (thijs-obj) [reply blog/2022-03-17 Release v0.0.14 Extended Bach date-time operations.md:26]: Yes, this makes sense :+1:

File: .htaccess-production (hunk 1 above)
Comment (ivarpruijn) [reply .htaccess-production:121]: OK, let's redirect them all to `docs/modeling/bach/api-reference/Series/`. Option 1 is a no-go, we should never have broken links, as they can be referenced from many places (old docker containers, social media, etc.) or bookmarked by people.

**Pull request merged**